src/include/wiredtiger.in

```diff
@@ -739,40 +739,40 @@ struct __wt_cursor {
 	WT_ITEM lower_bound, upper_bound;
 
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
-#define	WT_CURSTD_APPEND		0x000000001ull
-#define	WT_CURSTD_BULK			0x000000002ull
```

Comment:
This flag seems unaligned from the rest, is that automatically generated?

Reply:
only the flag is automatically generated - as we are only concerned about the character after the #define as it is integral to sorting the rest was not touched